docs/tutorial.md

```diff
@@ -516,25 +516,23 @@ tutorial-pipeline-run-1-deploy-web-jjf2l           deploy-web           4 hours
 tutorial-pipeline-run-1-build-skaffold-web-7jgjh   build-skaffold-web   4 hours ago   1 minute     Succeeded
 ```
 
-The status of type `Succeeded` shows the `Pipeline` ran successfully, also
-the status of individual Task runs are shown.
+The above output shows the status of `Pipeline` is `Succeeded`.  It also represents the status of each Task.
```

Comment:
These documents lines are kept short intentionally.  I don't know if there's a specific lint rule about it but I think we should maintain the line lengths unless this is going to be a global change with a supporting lint rule with.